model/cart.model.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const CartSchema = new mongoose.Schema({
+  cartItems: [
+    {
+      product: { type: mongoose.Schema.Types.ObjectId, ref: "Product" },
+      quantity: Number,
+      productID: Number,
+    },
+  ],
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+});
+
+const Cart = mongoose.model("Cart", CartSchema);
```

Comment:
- You can extract the `Schema` and `model` out of `mongoose` like 
```
const {Schema,model} = require("mongoose");
```
- But it is just a suggestion.